### PR TITLE
6.5 workbench release notes - initial revisions

### PR DESCRIPTION
**Pull request by jhaigh0**
**Description of work.**

- Moved notes from #34339 to main file
- Some grammar changes and fullstop additions
- Added more refferences to other documentation where possible

**To test:**

- Remove build/docs/doctrees
- Build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.5.0/mantidworkbench.rst [hunk 1]
```diff
@@ -12,14 +12,12 @@ New Features
 
 Bugfixes
 --------
-- Scaling for waterfall plots has been fixed for scaling in log scale plots
+- Scaling for waterfall plots has been fixed for scaling in log scale plots.
```
Comment (RichardWaiteSTFC) [inline docs/source/release/v6.5.0/mantidworkbench.rst:15]: Perhaps this is clearer?
`Log-scaling has been fixed in waterfall plots.`

**Review by RichardWaiteSTFC — CHANGES_REQUESTED** [review]
Small re-wording suggestion.

It's not a big deal but does it matter that there are still headings for empty sections (e.g. New Features in sliceviewer) - or is that sorted out at the final pass through the notes (or not sorted out at all).

**Comment by sf1919** [comment]
I can confirm that the release notes build correctly and links work correctly - @RichardWaiteSTFC over to you

**Review by RichardWaiteSTFC — APPROVED** [review]
Thanks @sf1919 for building this! I'm happy to approve!

**Pull request merged**